cclient/serialization/json/json_serializer.c

```diff
@@ -4,6 +4,20 @@
 #include "cJSON.h"
 #include "json_serializer.h"
 
+int address_to_flex_trit(address_t addr, const char* trytes) {
```

Comment:
this method looks odd. can't we just adjust the address_t struct to contain flex_trits direclty?

Reply:
address_t is a point of trit_array_t, regard with memory model we need to store extra info, say num_trits and num_bytes. 
@ifullgaz and @tsvisabo, do you have any suggestions about converting flex_trits from a string(trytes)? 
actually for deserialization here should check the num_trits and num_bytes that are assigned by the user.